csrc/ops/arith.h

```diff
@@ -120,6 +120,12 @@ NVF_API TensorView* reductionOpRaw(
     bool keep_dim = false,
     DataType dtype = DataType::Null);
 
+struct ScaledTensorView {
+  TensorView* mat;
```

Comment:
```suggestion
  TensorView* tv;
```
Looks general enough to be used beyond matrix